docs/runbooks/CnaoNotReady.md

```diff
@@ -0,0 +1,28 @@
+# CnaoNotReady
+
+## Meaning
+
+Cluster-network-addons-operator (CNAO) deploys additional networking components on top of the cluster.
```

Comment:
I am not sure we need to add the CNAO is, looks like the alarm is happening when "additional networking components" are deployed.

They don't explain what kubevirt is here https://github.com/kubevirt/monitoring/blob/main/docs/runbooks/KubeVirtComponentExceedsRequestedCPU.md#meaning

Maybe we can talk about CNAO CR

```
This alert fires when CNAO CR is not ready, which means that one of the deployed components is not ready.

Reply:
> They don't explain what kubevirt is here https://github.com/kubevirt/monitoring/blob/main/docs/runbooks/KubeVirtComponentExceedsRequestedCPU.md#meaning

but they do explain it on an HCO alert [KubevirtHyperconvergedClusterOperatorUSModification](https://github.com/kubevirt/monitoring/blob/main/docs/runbooks/KubevirtHyperconvergedClusterOperatorUSModification.md) :) 

But I like your line about the CNAO CR, will change that, but let's continue to discuss if the CNAO intro thing is needed..

Reply:
As discussed on #18 , will keep this part.